### PR TITLE
Create a dedicated nodepool for OHW event

### PR DESCRIPTION
**Pull request by sgibson91**
ref: https://github.com/2i2c-org/infrastructure/issues/2879

I've set the minimum to 0 for now. I can scale it up Monday AM before the start of the event.

### CODE REVIEW

**Comment by sgibson91** [comment]
<details> <summary> Output of `tf plan` </summary>
```
Terraform will perform the following actions:

  # google_container_node_pool.notebook["ohw"] will be created
  + resource "google_container_node_pool" "notebook" {
      + cluster                     = "pilot-hubs-cluster"
      + id                          = (known after apply)
      + initial_node_count          = 0
      + instance_group_urls         = (known after apply)
      + location                    = "us-central1-b"
      + managed_instance_group_urls = (known after apply)
      + max_pods_per_node           = (known after apply)
      + name                        = "nb-ohw"
      + name_prefix                 = (known after apply)
      + node_count                  = (known after apply)
      + node_locations              = (known after apply)
      + operation                   = (known after apply)
      + project                     = "two-eye-two-see"
      + version                     = (known after apply)

      + autoscaling {
          + location_policy = (known after apply)
          + max_node_count  = 100
          + min_node_count  = 0
        }

      + management {
          + auto_repair  = true
          + auto_upgrade = false
        }

      + node_config {
          + disk_size_gb      = (known after apply)
          + disk_type         = "pd-balanced"
          + guest_accelerator = (known after apply)
          + image_type        = (known after apply)
          + labels            = {
              + "2i2c.org/community"           = "ohw"
              + "hub.jupyter.org/node-purpose" = "user"
              + "k8s.dask.org/node-purpose"    = "scheduler"
            }
          + local_ssd_count   = (known after apply)
          + logging_variant   = "DEFAULT"
          + machine_type      = "n1-highmem-16"
          + metadata          = (known after apply)
          + min_cpu_platform  = (known after apply)
          + oauth_scopes      = [
              + "https://www.googleapis.com/auth/cloud-platform",
            ]
          + preemptible       = false
          + resource_labels   = {
              + "community" = "ohw"
            }
          + service_account   = "pilot-hubs-cluster-sa@two-eye-two-see.iam.gserviceaccount.com"
          + spot              = false
          + tags              = []
          + taint             = [
              + {
                  + effect = "NO_SCHEDULE"
                  + key    = "hub.jupyter.org_dedicated"
                  + value  = "user"
                },
              + {
                  + effect = "NO_SCHEDULE"
                  + key    = "2i2c.org/community"
                  + value  = "ohw"
                },
            ]

          + workload_metadata_config {
              + mode = "GKE_METADATA"
            }
        }
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Changes to Outputs:
  ~ regular_channel_latest_k8s_versions = {
      ~ "1."    = "1.27.2-gke.1200" -> "1.27.3-gke.100"
      ~ "1.24." = "1.24.14-gke.1200" -> "1.24.14-gke.2700"
      ~ "1.25." = "1.25.10-gke.1200" -> "1.25.10-gke.2700"
      ~ "1.26." = "1.26.5-gke.1200" -> "1.26.5-gke.2700"
      ~ "1.27." = "1.27.2-gke.1200" -> "1.27.3-gke.100"
    }
```
</details>

File: terraform/gcp/projects/pilot-hubs.tfvars [hunk 1]
```diff
@@ -53,6 +53,24 @@ notebook_nodes = {
     resource_labels : {
       "community" : "neurohackademy"
     },
+  },
+  # Nodepool for OceanHackWeek. Tracking issue: https://github.com/2i2c-org/infrastructure/issues/2879
+  "ohw" : {
+    # We expect around 100 users
+    min : 0,
+    max : 100,
+    machine_type : "n1-highmem-16",
```
Comment (consideRatio) [inline terraform/gcp/projects/pilot-hubs.tfvars:62]:
```suggestion
    machine_type : "n2-highmem-16",
```

I opened a dedicated issue https://github.com/2i2c-org/infrastructure/issues/2923 suggesting we transition to `n2` from `n1`, if you haven't already applied this I suggest going for `n2` right away - there is quota for both `n1` and `n2` in all GCP projects now.

**Review by consideRatio — APPROVED** [review]

File: terraform/gcp/projects/pilot-hubs.tfvars (hunk 1 above)
Comment (sgibson91) [reply terraform/gcp/projects/pilot-hubs.tfvars:62]: Ah yes, because the last time we had this conversation there was no quota left :D

**Comment by sgibson91** [comment]
Applied successfully!

File: terraform/gcp/projects/pilot-hubs.tfvars (hunk 1 above)
Comment (consideRatio) [reply terraform/gcp/projects/pilot-hubs.tfvars]: Wiee nice!!!

**Pull request merged**